pkg/apps/application-list.jsx

```diff
@@ -47,7 +47,11 @@ const ApplicationRow = ({ comp, progress, progress_title, action }) => {
 
     let summary_or_progress;
     if (progress) {
-        summary_or_progress = <ProgressBar title={progress_title} data={progress} />;
+        summary_or_progress = (
+            <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+                <span id={progress_title} className="progress-title-span">{progress_title}</span>
```

Comment:
id's needs to be unique per document. progress title is something like 'removing'. You need to make it more specific.

How about {comp.name + "-progress"}